pyani_plus/private_cli.py

```diff
@@ -349,20 +349,10 @@ def log_comparison(  # noqa: PLR0913
     config_id: REQ_ARG_TYPE_CONFIG_ID,
     query_fasta: REQ_ARG_TYPE_QUERY_FASTA,
     subject_fasta: REQ_ARG_TYPE_SUBJECT_FASTA,
-    identity: Annotated[
-        float,
-        typer.Option(
-            help="Percent identity",
-            show_default=False,
-            min=0.0,
-            max=1.0,
-        ),
-    ],
-    aln_length: Annotated[
-        int, typer.Option(help="Alignment length", show_default=False)
-    ],
     *,
     # Optional comparison table entries
+    aln_length: Annotated[int | None, typer.Option(help="Alignment length")] = None,
+    identity: Annotated[float | None, typer.Option(help="Percent identity")] = None,
```

Comment:
I don't know typer well enough to see a better solution. Not a big issue for an internal API.